script/test

```diff
@@ -1,24 +1,22 @@
 #!/bin/bash
 
 COMMAND=$1
-PROJECT_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )"
-ROVER="/usr/local/openresty/luajit/bin/rover"
+PROJECT_PATH=${PROJECT_PATH-$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )}
+ROVER=${ROVER-/usr/local/openresty/luajit/bin/rover}
 
 function run_busted_tests {
     sudo cp -R /tmp/lua_modules .
 
-    cd "${PROJECT_PATH}" || exit; ${ROVER} exec ./bin/busted
+    cd "${PROJECT_PATH}" || exit; \
+        TEST_NGINX_REDIS_HOST=redis ${ROVER} exec ./bin/busted
 }
 
 function run_nginx_unit_tests {
     cd "${PROJECT_PATH}" || exit
 
     sudo cp -R /tmp/lua_modules .
 
-    sudo TEST_NGINX_APICAST_PATH="${PROJECT_PATH}/gateway" \
-         TEST_NGINX_BINARY=openresty \
-         TEST_NGINX_REDIS_HOST=redis \
-         ${ROVER} exec prove
+    sudo -E "${ROVER}" exec prove
```

Comment:
I'm not sure. Then I'd need to run `sudo -E script/test` every time instead of just `script/test`.
Another solution would be to modify the image in a way that this does not require `sudo`.

Reply:
`make development` can run the docker with `-u root` and then there is no need for sudo.
If we aim for this to be executed by users then it surely can't use sudo.